app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposer.kt

```diff
@@ -111,71 +115,64 @@ fun MessageComposer(
                 .collect { onMentionMember(it) }
         }
 
+        LaunchedEffect(keyboardHeight) {
+            messageComposerState.keyboardHeight = keyboardHeight
+        }
+
+        LaunchedEffect(isKeyboardVisible) {
+            if (!isKeyboardVisible && !messageComposerState.attachmentOptionsDisplayed) {
+                messageComposerState.toEnabled()
+                messageComposerState.focusManager.clearFocus()
+                messageComposerState.cancelReply()
```

Comment:
do we really want to cancel reply when user just hides a keyboard?

Reply:
good question 🤔

Reply:
it's removed, it shouldn't cancel the reply